src/ReachSets/discretize.jl

```diff
@@ -887,3 +887,14 @@ function _discretize_interval_matrix_inhomog(U, Ω0_homog, linear_maps,
     end
     return Ω0, Ud
 end
+
+# fallback implementation for conversion (if applicable) or overapproximation
+function _convert_or_overapproximate(X::Type{<:Zonotope}, Y::LazySet)
```

Comment:
We can be more general here.
```suggestion
function _convert_or_overapproximate(X::Type{<:AbstractPolytope}, Y::LazySet)
```